src/main/java/br/com/sbk/sbking/networking/server/SBKingServer.java

```diff
@@ -0,0 +1,173 @@
+  private Table table;
+  private Map<UUID, Direction> playerDirections = new HashMap<UUID, Direction>();
+
+  public SBKingServer(Table table) {
+    this.table = table;
+  }
+
+  public void setKryonetSBKingServer(KryonetSBKingServer kryonetSBKingServer) {
+    this.kryonetSBKingServer = kryonetSBKingServer;
+  }
+
+  public void addUnnammedPlayer(UUID identifier) {
+    Player player = new Player(identifier, "SBKingServer Unnamed");
+    this.identifierToPlayerMap.put(identifier, player);
+  }
+
+  private Direction getDirectionFromIdentifier(UUID playerIdentifier) {
+    return playerDirections.get(playerIdentifier);
+  }
+
+  public void play(Card card, UUID playerIdentifier) {
+    Direction from = this.getDirectionFromIdentifier(playerIdentifier);
+    this.table.getGameServer().notifyPlayCard(card, from); // FIXME Law of Demeter
+  }
+
+  public void moveToSeat(Direction direction, UUID playerIdentifier) {
+    Direction from = this.getDirectionFromIdentifier(playerIdentifier);
+    Direction to = direction;
+    LOGGER.debug("Moving from: " + from + "  to: " + to);
+    this.table.moveToSeat(playerIdentifier, to);
+    // Should also remove the previously seated player from the map
+    if (from != null && from.equals(to)) {
+      playerDirections.remove(playerIdentifier);
+    } else {
+      playerDirections.put(playerIdentifier, to);
+    }
+  }
+
+  public void choosePositive(UUID playerIdentifier) {
+    KingGameServer kingGameServer = (KingGameServer) this.table.getGameServer();
+    PositiveOrNegative positiveOrNegative = new PositiveOrNegative();
+    positiveOrNegative.setPositive();
+    kingGameServer.notifyChoosePositiveOrNegative(positiveOrNegative,
+        this.getDirectionFromIdentifier(playerIdentifier));
+  }
+
+  public void chooseNegative(UUID playerIdentifier) {
+    KingGameServer kingGameServer = (KingGameServer) this.table.getGameServer();
+    PositiveOrNegative positiveOrNegative = new PositiveOrNegative();
+    positiveOrNegative.setNegative();
+    kingGameServer.notifyChoosePositiveOrNegative(positiveOrNegative,
+        this.getDirectionFromIdentifier(playerIdentifier));
+  }
+
+  public void chooseGameModeOrStrain(String gameModeOrStrainString, UUID playerIdentifier) {
+    GameServer gameServer = this.table.getGameServer();
+    Ruleset gameModeOrStrain = RulesetFromShortDescriptionIdentifier.identify(gameModeOrStrainString);
+    Direction directionFromIdentifier = this.getDirectionFromIdentifier(playerIdentifier);
+    if (gameModeOrStrain != null && directionFromIdentifier != null) {
+      if (gameServer instanceof MinibridgeGameServer) {
+        MinibridgeGameServer minibridgeGameServer = (MinibridgeGameServer) this.table.getGameServer();
+        minibridgeGameServer.notifyChooseGameModeOrStrain(gameModeOrStrain, directionFromIdentifier);
+      } else if (gameServer instanceof KingGameServer) {
+        KingGameServer minibridgeGameServer = (KingGameServer) this.table.getGameServer();
```

Comment:
Here: `KingGameServer minibridgeGameServer`. It's best to rename the variable to `kingGameServer`.

Optionally, since this `else if` contains the same piece of code as the `if` block above, we could do something more polymorphic if you are feeling it's worth it

Reply:
Damned copy/paste! I will fixed the naming.
And for the polymorphsim, I leave as it is for now, because those classes do not share a semantic ancestor yet. The `GameServer`s were rushed and probably will too will refactored after `Table`.